libs/sdk-model/src/dashboard/filterContext.ts

```diff
@@ -71,6 +71,11 @@ export interface IDashboardAttributeFilter {
          * Parent filters that are limiting elements available in this filter
          */
         filterElementsBy?: IDashboardAttributeFilterParent[];
+
+        /**
+         * Custom title of the filter.
```

Comment:
I think there should be more information about it (e.g., it has priority over the attribute title, etc...)